fdbclient/MultiVersionTransaction.actor.cpp

```diff
@@ -1143,12 +1185,28 @@ void MultiVersionDatabase::DatabaseState::updateDatabase(Reference<IDatabase> ne
 			versionMonitorDb = db;
 		} else {
 			// For older clients that don't have an API to get the protocol version, we have to monitor it locally
-			versionMonitorDb = MultiVersionApi::api->getLocalClient()->api->createDatabase(clusterFilePath.c_str());
+			try {
+				versionMonitorDb = MultiVersionApi::api->getLocalClient()->api->createDatabase(clusterFilePath.c_str());
+			} catch (Error& e) {
+				// We can't create a database to monitor the cluster version. This means we will continue using the
+				// previous one, and that could result in us having extra connections
```

Comment:
Maybe this doesn't actually cause us to have extra connections. We'll just be using a different version for the version monitor than the local DB.